x-pack/plugins/actions/server/monitoring/register_cluster_collector.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import stats from 'stats-lite';
+import { MonitoringCollectionSetup } from '../../../monitoring_collection/server';
+import { CoreSetup } from '../../../../../src/core/server';
+import { ActionsPluginsStart } from '../plugin';
+import { ClusterActionsMetric } from './types';
+
+export function registerClusterCollector({
+  monitoringCollection,
+  core,
+}: {
+  monitoringCollection: MonitoringCollectionSetup;
+  core: CoreSetup<ActionsPluginsStart, unknown>;
+}) {
+  monitoringCollection.registerMetric({
+    type: 'cluster_actions',
+    schema: {
+      overdue: {
+        count: {
+          type: 'long',
+        },
+        duration: {
```

Comment:
nit: `duration` in the context of `overdue` is a little confusing to me, maybe because I'm always thinking about execution duration, so it took a couple of reads for me to realize it means the length of time the task is overdue by.

Reply:
Hmm. Do you think we should rename or add a clarifying comment?

Reply:
I want to say rename, but I can't come up with any better options so if no one else finds it confusing we can leave as is :)